melodies_monet/tests/test_get_data_cli.py

```diff
@@ -7,10 +7,26 @@
 import subprocess
 
 import numpy as np
+import pytest
+import requests
 import xarray as xr
 
 from melodies_monet.tutorial import fetch_example
 
+try:
+    requests.head("https://www1.ncdc.noaa.gov/pub/data/noaa/")
+except Exception:
+    ish_reachable = False
+else:
+    ish_reachable = True
+
+try:
+    r = requests.head("https://www1.ncdc.noaa.gov/pub/data/noaa/isd-lite/")
```

Comment:
Don't need to store.
```suggestion
    requests.head("https://www1.ncdc.noaa.gov/pub/data/noaa/isd-lite/")
```